tools/scripts/params/shared_all.json

```diff
@@ -5,12 +5,14 @@
         "@type": "/poktroll.shared.MsgUpdateParams",
         "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
         "params": {
-          "num_blocks_per_session": "4",
+          "num_blocks_per_session": "10",
```

Comment:
This can have impact on E2E tests. Dangerous to just change in the middle of this large PR.

Reply:
This is not used anywhere except for a single makefile target (`params_update_shared_all`) [1]. 

My intention here was to updating it to match the (newer) default. These JSON files exist to support their respective makefile targets and to serve as a reference. They've historically contained the default values, which is convenient when debugging new governance params or their E2E tests (i.e. easy reset).

![image](https://github.com/user-attachments/assets/d6748a79-5b38-467c-ace6-0cfcf1ece9ca)
> [1] Search results for the regex `params/.+/\.json`